Info/VtxInformation.h

```diff
@@ -31,12 +31,14 @@ class VtxInformation : public G4VUserPrimaryVertexInformation
   void SetThreeGammaGen(G4bool tf) {fThreeGammaGen = tf;};
   void SetTwoGammaGen(G4bool tf) {fTwoGammaGen = tf;};
   void SetPromptGammaGen(G4bool tf) {fPromptGammaGen = tf;};
+  void SetCosmicGen(G4bool isCosmic) { fCosmicGen = isCosmic; };
   void SetRunNr(G4int x) {fnRun = x;};
   void SetVtxPosition(G4double x, G4double y, G4double z);
   void SetLifetime(G4double x) {fLifetime = x;};
   G4bool GetThreeGammaGen() {return fThreeGammaGen;};
   G4bool GetTwoGammaGen() {return fTwoGammaGen;};
   G4bool GetPromptGammaGen() {return fPromptGammaGen;};
+  G4bool GetCosmicGammaGen() { return fCosmicGen; };
```

Comment:
Could you add const for those Get methods?

Reply:
done